cmd/varint/varint.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"encoding/binary"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+)
+
+var signed bool
+
+func main() {
+	log.SetFlags(0)
+	log.SetPrefix("varint: ")
+
+	flag.BoolVar(&signed, "s", false, "signed")
+	flagNotBoolVar(&signed, "u", true, "unsigned (default)")
+	flag.Parse()
```

Comment:
Will `flag.Parse` and `flag.Args` do the right thing when the first non-flag argument starts with a minus sign (i.e., a negative number to encode)?

I don't know if it does or not, but it's worth checking.